FiveCalls/FiveCalls/IssueDetailViewController.swift

```diff
@@ -44,8 +44,12 @@ class IssueDetailViewController : UIViewController, IssueShareable {
             tableView.deselectRow(at: indexPath, animated: true)
         }
         logs = ContactLogs.load()
-        
-        navigationController?.setNavigationBarHidden(false, animated: true)
+
+        if let nav = self.navigationController, nav.viewControllers.count > 1 {
+            navigationController?.setNavigationBarHidden(false, animated: true)
+        } else {
+            tableView.contentInset = UIEdgeInsets(top: 64, left: 0, bottom: 0, right: 0)
```

Comment:
any idea why the table view isn't picking up the content inset properly from the nav bar? In theory, you shouldn't have to hard-code this.